modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase

```diff
@@ -0,0 +1,92 @@
+@component-name = "portal-object"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Object";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-182586 - Verify it's possible to define output fields for my validation as partial validation"
```

Comment:
Verify an error message is seen for an invalid field in an object entry if the validation is set to partial validation